### ISSUE TITLE
Any way to do nested dropdown?

### ISSUE THREAD

**Issue by musaffa** [cibernox/ember-basic-dropdown#165]
I'm trying to do nested dropdown with this plugin. For nesting, I need to load a dropdown to the side of an element of the parent dropdown. Any idea how can this be done?

**Comment by cibernox** [comment]
This example works?
https://github.com/cibernox/ember-basic-dropdown/blob/master/tests/dummy/app/templates/index.hbs#L69-L85

**Comment by musaffa** [comment]
@cibernox In that example, the nested dropdown opens below the parent dropdown. Is there any option to open it to the left/right side of the parent dropdown?

**Comment by cibernox** [comment]
At the moment the only positioning strategies allowed on the component are `above/below` and horizotanlly it can be "anchored" to the right or to the left, but it cannot float on the right or to the left of the trigger.

However, since yesterday, users can roll out their own custom positioning strategy, passing `calculatePosition=myFunction`.

In the readme you have the signature of the function. In your case I think that implementing that positioning strategy would be quite simple:

```js
function(trigger, dropdown) {
  let { left: triggerLeft, top: triggerTop, width: triggerWidth } = trigger.getBoundingClientRect();
  return { style: { top, left: left + width } };
}
```

**Comment by musaffa** [comment]
I tested this in the example of nested dropdown. I had to change the function to this:

```Javascript
  function(trigger, dropdown) {
    let { left, top, width } = trigger.getBoundingClientRect();
    return { style: { top: top + 'px', left: (left + width) + 'px' } };
  }
```
Change 1: getBoundingClientRect returns an object containing left, top etc. It doesn't return any `triggerLeft`, `triggerTop`.

Change 2: inline style requires `px` otherwise it doesn't work.

Here is the result:

![screenshot from 2016-10-07 12-59-09](https://cloud.githubusercontent.com/assets/294616/19181318/1ada7dee-8c8e-11e6-8ad3-3462e11e1060.png)
